tests/Api/CustomersTest.php

```diff
@@ -26,10 +26,10 @@ public function test_can_call_magento_api_customers_all_with_filter()
     {
         Http::fake();
 
-        $api = MagentoFacade::api('customers')->all(1, 1, [
+        $api = MagentoFacade::api('customers')->all([
```

Comment:
@developh 
Did the order of arguments to `Customers::all()` change?

Reply:
@developh 
Please discard the changes to this test because the order of arguments for `Customers::all()` was reverted in `main` to remain compatible with the original package and therefore this test will fail if merged into `main`.
The same is true for `OrdersTest` and `ProductsTest`.